recipes/libsndfile/all/conanfile.py

```diff
@@ -16,15 +17,18 @@ class LibsndfileConan(ConanFile):
         "programs": [True, False],
         "experimental": [True, False],
         "with_alsa": [True, False],
-        "with_sqlite": [True, False],
+        "with_sqlite": ["deprecated", True, False],
+        "with_external_libs": [True, False],
```

Comment:
Could these be controlled individually?

Reply:
No, `ENABLE_EXTERNAL_LIBS` needs all.
https://github.com/libsndfile/libsndfile/blob/91128fcd60ab13cca0b1475960c75b07cdca7b04/CMakeLists.txt#L79